src/components/inline_edit/inline_edit_form.test.tsx

```diff
@@ -88,6 +88,32 @@ describe('EuiInlineEditForm', () => {
       expect(container.firstChild).toMatchSnapshot();
       expect(getByText('This is a placeholder.')).toBeTruthy();
     });
+
+    test('title for defaultValue', () => {
```

Comment:
See above comment - I think we can simplify down to just one test
```suggestion
    it('renders the read mode value in a title tooltip', () => {
```